### DISCUSSION TITLE
Are the `pnpm docs` already existing pnpm commands?

### DISCUSSION THREAD

**Discussion by btea** [pnpm/pnpm#6446] [Q&A] [answered]
When I execute `pnpm docs`, it will jump to the npm official website instead of executing the corresponding script. 🤔

**Comment by lvqq** [comment, marked as answer]
Yes, it's. And it will be passed to npm directly. See [pnpm](https://github.com/pnpm/pnpm/blob/main/pnpm/src/pnpm.ts#L28~L59). For the command same with keyword, you can try `pnpm run docs`

**Reply by btea** [reply]
I see, thank you for your reply 🙇‍♂️

**Comment by btea** [comment]
@lvqq Hello, is there any configuration to make `pnpm docs` have the same effect as `pnpm run docs`? 🤔

**Reply by lvqq** [reply]
Maybe a short alias could help?

**Reply by btea** [reply]
I checked the documentation, and it seems that on the windows system, only the system files can be modified to set the alias.

**Reply by lvqq** [reply]
It depends on which terminal ur using. For the repo [pnpm-alias](https://github.com/lvqq/pnpm-alias) hosted by myself, it supports cmd and powershell on Windows. But it doesn't support alias `pnpm docs` for now🤣

**Reply by btea** [reply]
Thanks, it looks good, but on windows you have to modify the powershell config file after all 🤣

Is it possible to implement a more convenient way in the future, such as directly creating an `.alias.cjs` file to achieve this?🤔
